guides/doc-Provisioning_Guide/topics/Cloud-Amazon_EC2.adoc

```diff
@@ -1,25 +1,38 @@
 
 The requirements for Amazon EC2 provisioning include:
 
+ifeval::["{Build}" == "foreman"]
+  * The installation media that you require for the operating systems you want to use to provision using Amazon EC2.
+endif::[]
+ifeval::["{Build}" == "satellite"]
   * Synchronized content repositories for Red{nbsp}Hat Enterprise Linux 7. For more information, see link:/html/content_management_guide/importing_red_hat_content#Importing_Red_Hat_Content-Synchronizing_Red_Hat_Repositories[Synchronizing Red{nbsp}Hat Repositories] in the _Content Management Guide_.
+endif::[]
   * A {SmartProxyServer} managing a network in your EC2 environment. Use a Virtual Private Cloud (VPC) to ensure a secure network between the hosts and the {SmartProxyServer}.
   * An Amazon Machine Image (AMI) for image-based provisioning.
+ifeval::["{Build}" == "satellite"]
   * An activation key for host registration. For more information, see link:/html/content_management_guide/managing_activation_keys#Managing_Activation_Keys-Creating_an_Activation_Key[Creating An Activation Key] in the _Content Management_ guide.
+endif::[]
+ifeval::["{Build}" == "foreman"]
+  * If the Katello plugin is installed, an activation key for host registration. For more information, see link:/html/content_management_guide/managing_activation_keys#Managing_Activation_Keys-Creating_an_Activation_Key[Creating An Activation Key] in the _Content Management_ guide.
+endif::[]
 
 [[Provisioning_Cloud_Instances_in_Amazon_EC2-Adding_a_Amazon_EC2_Connection_to_the_Satellite_Server]]
 === Adding an Amazon EC2 Connection to the {ProjectServer}
 
 Use this procedure to add the Amazon EC2 connection in the {ProjectServer}'s compute resources.
 
 .Time Settings and Amazon Web Services
-Amazon Web Services uses time settings as part of the authentication process. Ensure that {ProjectServer}'s time is correctly synchronized. Ensure that an NTP service, such as `ntpd` or `chronyd`, is running properly on the {ProjectServer}. Failure to provide the correct time to Amazon Web Services can lead to authentication failures. For more information about synchronizing time in {Project}, see link:link:/html/installing_satellite_server_from_a_connected_network/installing_satellite_server#synchronizing_time[Synchronizing Time] in _Installing {ProjectServer} from a Connected Network_.
+Amazon Web Services uses time settings as part of the authentication process. Ensure that {ProjectServer}'s time is correctly synchronized. Ensure that an NTP service, such as `ntpd` or `chronyd`, is running properly on the {ProjectServer}. Failure to provide the correct time to Amazon Web Services can lead to authentication failures.
+ifeval::["{Build}" == "satellite"]
+For more information about synchronizing time in {Project}, see link:link:/html/installing_satellite_server_from_a_connected_network/installing_satellite_server#synchronizing_time[Synchronizing Time] in _Installing {ProjectServer} from a Connected Network_.
+endif::[]
```

Comment:
These links are invalid now, but will be valid in the future. I'd not hide them.